### ISSUE TITLE
Rename "primary ..." and "secondary educational sector"

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1413]
I suggest to rename these fields to "main educational sector" and "further educational sector". There is the possibility of a misunderstanding because primary, secondary and tertiary education is widely used for gradelevels.
I think this user misunerstood the categories:
https://oerworldmap.org/resource/urn:uuid:0af1b987-ec31-4540-a262-0b38b4ca011b

**Comment by trugwaldsaenger** [comment]
Good point. On the other hand "further educational sector" could be misunderstood as well ("further education"). I think we should keep this point in mind and come back to it, when we review our experience with the educational sectors.

**Comment by karindr** [comment]
We can also use "main sector" and "secondary sector". The combination makes clear that we don't think of primary and secondary education here.

Here is the next user who misunderstood the naming:
https://oerworldmap.org/resource/urn:uuid:caf91d7d-5e9b-4665-a74b-3a38f1af60de

**Comment by trugwaldsaenger** [comment]
Another possibility would be "main sector" and "additional sector". @drrobertfarrow what do you think, any other suggestions?

Apart from finding the best suitable wording I just mentioned that we do not have help texts included. This might improve the situation as well. I propose something like this:

"Choose the educational sector of primary activity"

and

"Choose educational sector of additional activity"

One problem here is that the wording of the help text has to fit with all different data types. Maybe we could use "entry" as a kind of generic term for this:

"Choose the educational sector the entry primarily belongs to / is active in."

**Comment by karindr** [comment]
> On the other hand we should keep it simple, whenever we can. Translating one help sentence is much easier than translating 6 different help sentences...

When there are six help sentences crowdin will show you six sentences to translate. So I think we should think from the point of view of the user not form the point of view from the translator.
